services/api-server/src/simcore_service_api_server/services_rpc/wb_api_server.py

```diff
@@ -4,48 +4,178 @@
+)
 from servicelib.rabbitmq.rpc_interfaces.webserver.licenses.licensed_items import (
     get_licensed_items as _get_licensed_items,
 )
+from servicelib.rabbitmq.rpc_interfaces.webserver.licenses.licensed_items import (
+    release_licensed_item_for_wallet as _release_licensed_item_for_wallet,
+)
 
+from ..exceptions.backend_errors import (
+    CanNotCheckoutServiceIsNotRunningError,
+    InsufficientNumberOfSeatsError,
+    LicensedItemCheckoutNotFoundError,
+)
 from ..exceptions.service_errors_utils import service_exception_mapper
 from ..models.pagination import PaginationParams
-from ..models.schemas.model_adapter import LicensedItemGet
+from ..models.schemas.model_adapter import LicensedItemCheckoutGet, LicensedItemGet
 
 _exception_mapper = partial(service_exception_mapper, service_name="WebApiServer")
 
 
+def _create_licensed_items_get_page(
+    *, licensed_items_page: LicensedItemGetPage, page_params: PaginationParams
+) -> Page[LicensedItemGet]:
+    page = create_page(
+        [
+            LicensedItemGet(
+                licensed_item_id=elm.licensed_item_id,
+                name=elm.name,
+                license_key=elm.license_key,
+                licensed_resource_type=elm.licensed_resource_type,
+                pricing_plan_id=elm.pricing_plan_id,
+                created_at=elm.created_at,
+                modified_at=elm.modified_at,
+            )
+            for elm in licensed_items_page.items
+        ],
+        total=licensed_items_page.total,
+        params=page_params,
+    )
+    return cast(Page[LicensedItemGet], page)
+
+
 @dataclass
 class WbApiRpcClient:
     _client: RabbitMQRPCClient
 
     @_exception_mapper(rpc_exception_map={})
     async def get_licensed_items(
-        self, product_name: str, page_params: PaginationParams
+        self, *, product_name: str, page_params: PaginationParams
     ) -> Page[LicensedItemGet]:
         licensed_items_page = await _get_licensed_items(
             rabbitmq_rpc_client=self._client,
             product_name=product_name,
             offset=page_params.offset,
             limit=page_params.limit,
         )
-        page = create_page(
-            [
-                LicensedItemGet(
-                    licensed_item_id=elm.licensed_item_id,
-                    name=elm.name,
-                    licensed_resource_type=elm.licensed_resource_type,
-                    pricing_plan_id=elm.pricing_plan_id,
-                    created_at=elm.created_at,
-                    modified_at=elm.modified_at,
-                )
-                for elm in licensed_items_page.items
-            ],
-            total=licensed_items_page.total,
-            params=page_params,
-        )
-        return cast(Page[LicensedItemGet], page)
+        return _create_licensed_items_get_page(
+            licensed_items_page=licensed_items_page, page_params=page_params
+        )
+
+    @_exception_mapper(rpc_exception_map={})
```

Comment:
We need to review this together at some point. I would not like to diverge too much from the concept we are using in the webserver and here.